packages/portal/src/components/item/ItemMediaThumbnails.vue

```diff
@@ -1,32 +1,94 @@
 <template>
   <div
-    class="p-2"
+    class="d-flex flex-column position-relative"
   >
-    <h2>Thumbnails</h2>
     <div
-      class="d-flex overflow-auto"
+      ref="mediaThumbnails"
+      class="media-thumbnails d-flex flex-row flex-lg-column"
     >
-      <b-img-lazy
-        v-for="(image, index) in thumbnails"
+      <ItemMediaThumbnail
```

Comment:
We should eager load the first 5 thumbnails as they are always in view on page load.

Reply:
I'm not convinced about this. Since the thumbnails are always fixed sizes, there's no layout shift from having them lazy load.
While usually 5 thumbnails appear, this is somewhat dependent on the screen size.
Further there's an edge case where the user opens a link with a page already selected, in which case it becomes trickier to identify which thumbnails need to be loaded already.

Reply:
I see there are the grey squares reserving the space for the thumbnail images. And that is both when lazy or eager loaded, so doesn't seem to really make a difference. Let's leave it to lazy load for now.